include/alias.h

```diff
@@ -0,0 +1,8 @@
+#include "backend.h"
+
+#ifndef ALIAS_H_
+#define ALIAS_H_
+
+int replace_with_alias(struct _psh_command *cmd);
```

Comment:
I would advise to rename this function to `expand_alias`. "Replace" doesn't sound like the correct verb.